spec/models/froggo_team_membership_spec.rb

```diff
@@ -5,6 +5,11 @@
   let!(:organization) { create(:organization) }
   let!(:team) { create(:froggo_team, organization: organization) }
 
+  describe "validations" do
+    it { is_expected.to validate_numericality_of(:assignment_percentage)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/ImplicitSubject: Don't use implicit subject.

Reply:
mira esto! https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ImplicitSubject

sugieren hacer esto así:

`it { expect(subject).to validate_numerocality_of(...)`

ve si te funca